src/aws_client_app.erl

```diff
@@ -0,0 +1,27 @@
+%%%-------------------------------------------------------------------
+%% @doc aws_client public API
+%% @end
+%%%-------------------------------------------------------------------
+
+-module('aws_client_app').
+
+-behaviour(application).
+
+%% Application callbacks
+-export([start/2
+        ,stop/1]).
+
+%%====================================================================
+%% API
+%%====================================================================
+
+start(_StartType, _StartArgs) ->
+    'aws_client_sup':start_link().
```

Comment:
rebar3 quotes all atoms, it makes it easier if a user chose the name `MyGreatLib`.